components/common-go/experiments/flags.go

```diff
@@ -41,3 +42,7 @@ func SupervisorPersistServerAPIChannelWhenStart(ctx context.Context, client Clie
 func SupervisorUsePublicAPI(ctx context.Context, client Client, attributes Attributes) bool {
 	return client.GetBoolValue(ctx, SupervisorUsePublicAPIFlag, false, attributes)
 }
+
+func IsSetJavaXmx(ctx context.Context, client Client, attributes Attributes) bool {
+	return client.GetBoolValue(ctx, SetJavaXmxFlag, false, attributes)
```

Comment:
@svenefftinge default should be true, for backward compatibility? not sure

Reply:
I'd like to go the other way around and see how it's going. Ideally we can get rid of this special hack and users do this themselves.

Reply:
ok, it is just if there is a connection hiccups or so it maybe impossible to revert this change via the feature flag